src/bin/find.js

```diff
@@ -11,58 +10,59 @@ var options = {
   error: ['error'],
   nc: [],
   core: ['core'],
-  verbose: ['verbose', 'v'],
-}
+  verbose: ['verbose', 'v']
+};
 
-var argv = require('yargs')
+const argv = require('yargs')
   .boolean(Object.keys(options))
   .alias(options)
   .default('error', true)
   .default('core', true)
-  .argv
+  .argv;
 
-var getRuleURI = require('eslint-rule-documentation')
+const getRuleURI = require('eslint-rule-documentation');
+const getRuleFinder = require('../lib/rule-finder');
+const cli = require('../lib/cli-util');
 
-var cli = require('../lib/cli-util')
+let processExitCode = argv.u && (!argv.n || argv.error) ? 1 : 0;
```

Comment:
Why `let`? The fix of #199 is not yet merged, so in this context (without any reassignment later on), I think it should be `const`.